spec.bs

```diff
@@ -418,19 +418,29 @@ try {
     not) be used by a [=Relying Party=] for other authentication flows (e.g.,
     login).
 
-: <dfn>Steps to determine if a credential is SPC-enabled</dfn>
-:: An as-yet undefined process by which a user agent can, given a [=credential
-    ID=], determine if the credential represented by that ID is an [=SPC
-    Credential=].
+    Note: The current version of this specification requires the [=Relying
+          Party=] to explicitly opt in for a credential to be used in SPC in
+          either a first-party or third-party context. Longer-term, our
+          intention is that all [=public key credential|WebAuthn credentials=]
+          will be usable for SPC in a first-party context (e.g., on the
+          [=Relying Party's=] domain) and opt-in will only be required to allow
+          a credential to be used by a third-party.
+
+: <dfn>Steps to silently determine if a credential is SPC-enabled</dfn>
+:: An as-yet undefined process by which a user agent can, given a
+    [[webauthn-3#relying-party-identifier|Relying Party Identifier]] and a
+    [=credential ID=], silently (i.e., without user interaction) determine if
+    the credential represented by that ID is an [=SPC Credential=].
 
     NOTE: See <a href="https://github.com/w3c/webauthn/issues/1667">WebAuthn
     issue 1667</a>.
 
 : <dfn>Steps to silently determine if a credential is available for the current device</dfn>
-:: An as-yet undefined process by which a user agent can, given a [=credential
-    ID=], silently (i.e., without user interaction) determine if the credential
-    represented by that credential ID is available for the current device (i.e.,
-    could be successfully used as part of a WebAuthn
+:: An as-yet undefined process by which a user agent can, given a
+    [[webauthn-3#relying-party-identifier|Relying Party Identifier]] and a
+    [=credential ID=], silently (i.e., without user interaction) determine if
+    the credential represented by that credential ID is available for the
+    current device (i.e., could be successfully used as part of a WebAuthn
```

Comment:
I am wondering about the phrase "current device" here and elsewhere in the API. It might be overly constraining in light of synched credentials and CABLE. Would it be more inclusive to say "current API call"?

Reply:
Great question. I'm not sure how best to resolve that yet (maybe worth adding a minor comment to https://github.com/w3c/secure-payment-confirmation/issues/174), so going to leave for now.